packages/design-system/src/components/section-title.stories.tsx

```diff
@@ -0,0 +1,85 @@
+import { SectionTitle } from "./section-title";
+import { Text } from "./text";
+import { StoryGrid, StorySection } from "./storybook";
+import { useState, type ComponentProps } from "react";
+
+export default {
+  title: "Library/Section Title",
+};
+
+const Wrapped = ({
+  initialIsOpen = false,
+  ...props
+}: Omit<ComponentProps<typeof SectionTitle>, "onOpenChange" | "isOpen"> & {
+  initialIsOpen?: boolean;
+}) => {
+  const [isOpen, setOpen] = useState(initialIsOpen);
+  return (
+    <div style={{ display: "flex", alignItems: "center", gap: 8 }}>
+      <div style={{ width: 240, border: "dashed 3px #e3e3e3" }}>
+        <SectionTitle
+          isOpen={isOpen}
+          onOpenChange={setOpen}
+          hasItems={["local", "remote"]}
+          {...props}
```

Comment:
please don't use spreads especially in examples 
`onAdd` is it come from props or how?

Reply:
I don't see anything wrong with spread here. 

Intent here: `Wrapped` wraps `SectionTitle` with some code that otherwise would have been repeated a lot in demos, it controls some of the props, and passes through the rest. 

Spread is perfect to express: "passes through the rest".

Reply:
Talked in DM: I didn't know something like this is not a TS error:

```
const props = {invalidProp: "something"}
<Something foo="bar" {...props} />
```

I agree it's a good idea to avoid props spread when possible because of this.